scp.c

```diff
@@ -678,6 +679,7 @@ main(int argc, char **argv)
 	else {
 		if (targetshouldbedirectory)
 			verifydir(argv[argc - 1]);
+		// verify_file_exists(argv[argc - 1]);
```

Comment:
I wanna add the condition for tolocal but the tests fail so I add it with the comment line. I can delete the line if it breaks something.